python/mp/rosary.py

```diff
@@ -29,25 +30,54 @@ def copy_color(self, color):
         self.color = copy.copy(color)
 
 
+class Updater:
+    def __init__(self, name='', bead_list=[], osc_client=None):
+        self.name = name
+        self.bead_list = bead_list
+        self.osc_client=osc_client
+        
+    def update(self):
+        msg = osc_message_builder.OscMessageBuilder(address = "/bead")
+        msg.add_arg(self.name)                 # name (class)
+        msg.add_arg(int(0))                    # base
+        msg.add_arg(int(len(self.bead_list)))  # length
+        
+        payload = bytearray()
+        for bead in self.bead_list:
+            payload.extend(struct.pack('!HHHH',
+                                       int(bead.color.r * 0xffff),
+                                       int(bead.color.g * 0xffff),
+                                       int(bead.color.b * 0xffff),
+                                       int(bead.color.brightness)
+            ))
+
+        msg.add_arg(bytes(payload))
+
+        self.osc_client.send(msg.build())
+
+
```

Comment:
This object contains the context needed to send an OSC message for a set of beads. This context is name (class), the bead_list for the set of beads, and the osc_client object.